src/vs/workbench/workbench.common.main.ts

```diff
@@ -390,6 +390,8 @@ import 'sql/workbench/contrib/query/common/resultsGrid.contribution';
 // data explorer
 import 'sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution';
 import 'sql/workbench/contrib/dataExplorer/browser/nodeActions.common.contribution';
+import 'sql/workbench/contrib/dataExplorer/browser/extensions.contribution';
```

Comment:
Don't really like the name here TBH - way too generic and doesn't make sense with what's in the actual file.

Reply:
I'm up for any suggestions!

Reply:
Hm, actually thinking about it I'm guessing this is named this way because these are contributions FROM extensions. So it's probably fine as. Something like extensionMenuItems.contribution could also work since it's fairly unlikely this is going to get other extension points added.